components/post_view/embedded_bindings/select_binding/select_binding.tsx

```diff
@@ -72,19 +74,29 @@ class SelectBinding extends React.PureComponent<Props, State> {
             return;
         }
 
-        const {userId, post} = this.props;
+        const {post} = this.props;
 
-        const call = createCallRequest(
-            binding.call,
-            userId,
+        let teamID = '';
+        const {data} = await this.props.actions.getChannel(post.channel_id) as {data?: any; error?: any};
+        if (data) {
```

Comment:
`if (data) -> if(data & data is Channel)` ? (ensures the `data as Channel` below is successful)